src/stactools/cli/commands/validate.py

```diff
@@ -15,25 +17,42 @@ def create_validate_command(cli):
     @click.option("--recurse/--no-recurse",
                   default=True,
                   help=("If false, do not validate any children "
-                        "(only useful for Catalogs and Collections"))
+                        "(only useful for Catalogs and Collections)"))
     @click.option("--links/--no-links",
                   default=True,
                   help=("If false, do not check any of the objects's links."))
     @click.option(
         "--assets/--no-assets",
         default=True,
         help=("If false, do not check any of the collection's/item's assets."))
-    def validate_command(href, recurse, links, assets):
+    @click.option("--stac-vrt",
+                  is_flag=True,
+                  default=False,
+                  help=("If true, verify that the required fields exist "
+                        "for use by the stac-vrt tool. (default: false)"))
+    def validate_command(href, recurse, links, assets, stac_vrt):
         """Validates a STAC object.
 
         Prints any validation errors to stdout.
         """
         object = pystac.read_file(href)
 
-        if isinstance(object, Item):
-            errors = validate(object, None, False, links, assets)
-        else:
-            errors = validate(object, object, recurse, links, assets)
+        validators = [None]  # Start with the existing default
```

Comment:
I think we should convert all of our checks to validators to keep things consistent, i.e. `LinksValidator` and `AssetsValidator`.